src/hooks/useAchievementTracker.ts

```
@@ -0,0 +1,198 @@
+          shipName: data?.shipName,
+          shipsSunk: stats.shipsSunk,
+          carriersSunk: stats.carriersSunk,
+          destroyersSunk: stats.destroyersSunk,
+          submarinesSunk: stats.submarinesSunk,
+          cruisersSunk: stats.cruisersSunk,
+          battleshipsSunk: stats.battleshipsSunk,
+        });
+        break;
+
+      case GameEvents.USER_WIN:
+        const timeToWin = Date.now() - stats.startTime;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)